rst/topics/failover.rst

```diff
@@ -191,6 +217,13 @@ These are clusterwide parameters:
 * ``state_provider``: "tarantool" / "etcd".
 * ``tarantool_params``: ``{uri = "...", password = "..."}``.
 * ``etcd2_params``: ``{endpoints = {...}, prefix = "/", lock_delay = 10, username = "", password = ""}``.
+* ``fencing_enabled``: ``true`` / ``false``;
+* ``fencing_pause`` -- the period of performing the check;
+* ``fencing_timout`` -- time to actuate fencing after the check fails;
+* ``failover_timeout`` -- time to declare unresponsive member as dead
```

Comment:
I don't see any mentions of this option in code

Reply:
V